src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -699,4 +699,7 @@ Additional information: {1}</value>
   <data name="Prefer_predefined_type" xml:space="preserve">
     <value>Prefer predefined type</value>
   </data>
+  <data name="Connection_to_remote_host_has_been_lost_some_features_might_stop_working_or_start_working_in_proc_do_you_want_to_recover_remote_host" xml:space="preserve">
+    <value>Connection to remote host has been lost. some features might stop working or start working in proc. do you want to recover remote host?</value>
```

Comment:
I am following the pattern we used for low memory indicator. I will let @srivatsn and @MattGertz decide.

basically context here is that, we recognized service hub being disappeared. like we detected low memory situation.

@CyrusNajmabadi  opinion is either automatically recover or let them restart VS (this is exactly the approach we took for low memory initially)

what I did here is what we did for low memory indicator in update 3. we show info bar to user and turn features off, and show users a way to turn it back on.